### PR TITLE
ASHK_7G19 QC model

### PR DESCRIPTION
**Pull request by ndukler**

### CODE REVIEW

**Comment by ndukler** [comment]
Not sure why the model is not being tested, it does appear to be registering. @grahamgower

File: stdpopsim/qc/HomSap.py [hunk 1]
```diff
@@ -908,6 +908,101 @@ def __init__(self):
...
+            msprime.MassMigration(time=T_eu_as, source=2, dest=1, proportion=1),
+            # OOA
+            msprime.MassMigration(time=T_AF, source=1, dest=0, proportion=1),
+            # Ancestral size change, reset population size
+            msprime.PopulationParametersChange(time=T_growth, initial_size=N_ANC,
+                                               population_id=0)
+        ]
+
+        _species.get_demographic_model(
+            "AshkSub_7G19").register_qc(GladsteinAshkSubstructure())
```
Comment (grahamgower) [inline stdpopsim/qc/HomSap.py:1003]: This bit needs to be at the toplevel scope of the file.

**Comment by ndukler** [comment]
Read to merge. Closes #511

**Comment by grahamgower** [comment]
LGTM, thanks Noah!

**Pull request merged**